tox.ini

```diff
@@ -48,7 +48,7 @@ commands =
 [testenv:docs]
 envdir = .tox/docs
 deps =
-    -rrequirements-doc.txt
```

Comment:
🤣

Reply:
More like 😓 
Makes you question what's the last time someone ran `make docs`.